.reek

```diff
@@ -11,3 +11,5 @@ exclude_paths:
 "app/helpers":
   FeatureEnvy:
     enabled: false
+Attribute:
+  enabled: false
```

Comment:
mỗi khi sửa config gì liên quan đến framgia-ci thì a comment giải thích cho e nhé

Reply:
à a có comment nhưng quên chưa ấn nút comment review nên nó chưa xuất hiện.:
Mình thêm dòng này vì, reek báo lỗi với activation_toke là is a writable attribute
Mà activation_token cần writer ra và read từ đối tượng, nên nó phải là attr_accessor.
Write: khởi tạo khi đối trước khi khởi tạo đối tượng.
Read: khi xác thực
Khác với remember_token: nó được write trong class user, mà chỉ cần reader từ đối tượng

Reply:
hự, e đọc a viết e chả hiểu gì

Reply:
- Tại reek có cảnh báo is a writable attribute nên a có thêm dòng đó để tắt cảnh báo đó.
- Lý do reek có cảnh báo đó là vì nó nhận thấy activation_token chỉ cần phương thức getter, nhưng thực tế thì cần cả setter và getter.